website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md

```diff
@@ -7,13 +7,13 @@ id: "artifacts"
 
 When dbt Cloud runs dbt jobs, it generates and saves *artifacts*. These artifacts, like `manifest.json`, `catalog.json`, and `sources.json` are used to power different aspects of dbt Cloud, namely: [dbt Docs](documentation) and [source freshness reporting](cloud-snapshotting-source-freshness).
 
-From your Account Settings page, you can select specific jobs that build the "production" version of your documentation
+Artifacts are on the project level and can be configured in your project settings.
```

Comment:
Hmm, do you think we could rephrase this to better communicate the idea? I'm picturing something like:

> While every dbt Cloud job will produce artifacts, typically there is only _one_ production job for a given project. If you select a production job on the Project Settings page, dbt Cloud will render links to the production Documentation and Source Freshness artifacts generated for that project in the nav sidebar.